routers/api/v1/repo/tree.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"fmt"
+	"strings"
+
+	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/git"
+	"code.gitea.io/sdk/gitea"
+)
+
+func GetTree(ctx *context.APIContext) {
+	sha := ctx.Params("sha")
+	if len(sha) == 0 {
+		ctx.Error(400, "sha not provided", nil)
+		return
+	}
+	Tree := GetTreeBySHA(ctx, sha)
```

Comment:
I agree with @adelowo, please lowercase first letter for all local variables

Reply:
Done.

Reply:
https://github.com/go-gitea/gitea/pull/5403/files#diff-f0fb94da81e656bdc93dabff1e5077d2R23

Reply:
Whoops. Thanks!